src/sourdough-starter.md

```diff
@@ -0,0 +1,25 @@
+# Sour dough starter
+
+This recipe cointains instructions for sour dough starter needed for all sour dough breads. The starter can be done with either wheat flour or rye flour for rye breads. Prefer organic flours as they contain most wild yeasts.
+
+- ⏲️ Prep time: 5 min/day
+- ⏲️ Wait time: 4 days
+## Ingredients
+
+- Wheat or rye flour
+- Water
+
+## Directions
+
+1. Day 1: mix 80g of water to 50g of flour and put a conver on the container loosely. Store somewhere warm.
+2. Day 2: the starter should be a bit bubbly. Feed the starter with 80g of water and 50g of flour and store again with loosely covered. If the starter doesn't have bubbles yet, wait another day.
+3. Day 3-4: repeat the feeding process
+4. You can now use the starter for baking sourdough bread. Remember to not use all the starter at once and feed the remaining starter to always have it available.
+5. Storing the starter: store the starter in a closed container or jar in the fridge. The starter will survive almost indefinitely. After some time a vinegarish liquid forms on top of the starter. Don't be afraid of this, it is there to protect the starter from microbes. It can be poured away when you use the starter next time. 
```

Comment:
I would suggest to store the sourdough max 1 to 2 weeks in the fridge without feeding. If it should be stored longer it can also be stored in the freezer or dried as powder. Maybe you could add this information.